src/Forum/ForumServiceProvider.php

```diff
@@ -141,6 +147,25 @@ protected function populateRoutes(RouteCollection $routes)
         );
     }
 
+    protected function checkCustomLessFormatBeforeSaving()
+    {
+        $this->app->make('events')->listen(PrepareSerializedSetting::class, function (PrepareSerializedSetting $event) {
```

Comment:
Can you move this logic to a dedicated event handler, please? Similar to what we do [here](https://github.com/flarum/core/blob/4f688fc9a25da9bfefe86b6cb418e2ca2a754519/src/User/UserServiceProvider.php#L89).

I feel this service provider is too generic a place to handle this stuff.